tests/integration/test_fim/test_synchronization/test_response_timeout.py

```diff
@@ -78,11 +78,11 @@ def wait_agent_initial_scan(time_out=60):
         truncate_agent_log()
         start_time = datetime.now()
         while datetime.now() < start_time + timedelta(seconds=time_out):
-            ssh_stdin, ssh_stdout, ssh_stderr = ssh.exec_command("sudo cat /var/ossec/logs/ossec.log")
+            ssh_stdin, ssh_stdout, ssh_stderr = ssh.exec_command("sudo cat /var/ossec/logs/wazuh.log")
```

Comment:
Same here. We can improve this by removing the hardcoded path to the log file, importing the `LOG_FILE_PATH` variable and using it as follows:
```suggestion
            ssh_stdin, ssh_stdout, ssh_stderr = ssh.exec_command(f"sudo cat {LOG_FILE_PATH}")
```

Reply:
Fixed in https://github.com/wazuh/wazuh-qa/commit/20fb78d2a4218a84aa5ce2a4323ff3437686b23b